src/libraries/Common/tests/System/Net/Http/SchSendAuxRecordHttpTest.cs

```diff
@@ -41,8 +74,13 @@ public async Task HttpClient_ClientUsesAuxRecord_Ok()
                 int serverTotalBytesReceived = 0;
                 int serverChunks = 0;
 
+                CircularBuffer buffer = new CircularBuffer(4);
```

Comment:
nit: It would be ok to use `var` here instead of the explicit type. `var` is used when the type is obvious such as when doing a `new` or a cast operation.

i.e.

```c#
var buffer = new CircularBuffer(4);
```